steps/row-count.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+VIEWS_DB="${views_db}"
+MODEL_DB="${model_db}"
+TRANSFORM_DB="${transform_db}"
+TRANSACTIONAL_DB="${transactional_db}"
+METRICS_FILE_PATH=/var/log/hive/metrics.json
+S3_LOCATION="${data_location}"
+
+(
+    source /opt/emr/logging.sh
+    function log_wrapper_message() {
+        log_pdm_message "$1" "row-count.sh" "$$" "Running as: $USER"
+    }
+    log_wrapper_message "Running row-count.sh file"
+
+    db_names=($TRANSFORM_DB $TRANSACTIONAL_DB $MODEL_DB)
+    res1=$(aws s3 ls $S3_LOCATION/pdm-dataset/hive/external/uc_pdm_model.db/)
```

Comment:
is it possibe to use above placeholders for these db names? Because the path can be the same but database names can change

Reply:
ex: `$S3_LOCATION/pdm-dataset/hive/external/MODEL_DB.db/`